assets/quick-add-bulk.js

```diff
@@ -55,15 +53,15 @@ if (!customElements.get('quick-add-bulk')) {
 
       listenForActiveInput() {
         if (!this.classList.contains('hidden')) {
-          this.getInput().addEventListener('focusin', (event) => event.target.select());
+          this.getInput()?.addEventListener('focusin', (event) => event.target.select());
```

Comment:
Nit but to remove some repetition in a few of these areas, you could do something like:

```ts
// or just rename `getInput` as a getter
get input() {
  return this.getInput();
}

listenForActiveInput() {
  if (!this.classList.contains('hidden') && this.input) {
  this.input.addEventListener(...);
  }
}

// same thing for `listenForKeydown`
```